README.md

```diff
@@ -63,16 +63,22 @@ The project includes both unit and integration tests. The integration tests simu
 To run the unit tests:
 
 ```console
+# without logs
 $ cargo test
+
+# with logs
+$ cargo test -- --nocapture
 ```
 
 To run the integration tests, `--features integration-test` flag must be enabled. Run integration tests with:
```

Comment:
As we are adding the feature into default list so we don't need this line anymore.